minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/RunMiNiFi.java

```diff
@@ -228,6 +233,9 @@ public static void main(String[] args) throws IOException, InterruptedException
             case "env":
                 runMiNiFi.env();
                 break;
+            case "flowstatus":
+                runMiNiFi.statusReport(args[1]);
```

Comment:
May want to do a check on this and provide usage information if no args are provided